doc/manual/memory.rst

```diff
@@ -21,7 +21,7 @@ MNE-Python can read data on-demand using the ``preload`` option provided in :ref
     raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
     raw = io.read_raw_fif(raw_fname, preload=False)
 
-.. note:: Filtering does not work with ``preload=False``.
+.. note:: Filtering, resampling and dropping or selection channels does not work with ``preload=False``.
```

Comment:
this is a case where the Oxford comma really helps (also there is a typo)
"Filtering, resampling, and dropping or selecting channels ..."